### PR TITLE
Create portunus.csl

### PR DESCRIPTION
**Pull request by arvalia**
Hi @rmzelle,

I worked on _arvalia.csl_ style (see https://github.com/citation-style-language/styles/pull/3460 ) and I finally realized this new one: _portunus.csl_.
Documentation: http://www.arvalia.it/portunus/portunus-documentazione/
Open submissions: http://www.arvalia.it/scrivi-un-articolo/ and http://www.arvalia.it/invia-un-link/
Be kinder this time!
Antonello
Arvalia.it magazine

### CODE REVIEW

File: portunus.csl [hunk 1]
```diff
@@ -0,0 +1,133 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" default-locale="it-IT">
+  <info>
+    <title>Portunus</title>
+    <id>http://www.zotero.org/styles/portunus</id>
+    <link href="http://www.zotero.org/styles/portunus" rel="self"/>
```
Comment (POBrien333) [inline portunus.csl]: do you know the template style you used for this? Could you also link to that with rel="template"?

Comment (arvalia) [reply portunus.csl]: I started from "Nature" template, but it's now so much different to relate it...

**Comment by POBrien333** [comment]
If you look at the report it says:
"expected et-al-min (#{min}) to be greater than et-al-use-first (#{first})"

that means you can't set both values to the same number. et-al-use-first needs to be lower than the other.

As per the guidelines of portunus: "In caso di 5 o più autori è indicato solo il primo seguito da et al."

Therefore, set them as et-al-min="5" et-al-use-first="1"

**Comment by rmzelle** [comment]
@arvalia, @adam3smith and I have the same question here as for your Arvalia CSL style. It looks like Portunus is an online library, consisting of a list of references (http://www.arvalia.it/portunus/). Again, this doesn't seem to be a good fit for this style repository.

Now that you learned that you can use custom styles with ZotPress, can we close this pull request as well?

Sorry, we always try to be kind to all our style contributors, but we do curate what type of styles are permitted into the repository.

**Comment by arvalia** [comment]
@POBrien333 thank you!

I'll go now to set different et al. values now. I hope to success!

One question: how to estract a full Italian guideline of the csl as the phrase you showed? ("In caso di 5 o più autori è indicato solo il primo seguito da et al.")

**Comment by arvalia** [comment]
@rmzelle , @adam3smith 

please let open Portunus' pull some time more: I'm not an expert but I have the target to make a good job.

The problem is that I did'nt fount in the Repository an Italian-well formatted style. If I'd founded it, I'd used...

The structure I'm looking for is:

–	**Name group.** Bulleted Name and full Surname. Example: M. Rossi
–	_(Comma and space to separate blocks)_
–	**Title group.** If chapter: Title, in _Book-title_ comma separated. Ex.: Gallia, in _De bello Gallico_
–	_(Comma and space)_
–	**Editorial history group.** Publisher and date, on this following formats. If book: _Publisher_ (year) Ex.: _Mondadori_ (2018). If article: _Journal_, extended date (no commas). Ex .: _Corriere_ 20 maggio 2018
–	_(End dot for each item)_

It's all: option “et al.” set at 5, all the rest of the options exactly as the Nature.csl style.

I maked also other changes like this:
- Introduce the title (only for in special categories) by a prefix (Ex.: Video before a broadcast contents).
- n. v. and p. prefix for numbers, volumes and pages.

You can help me giving the time to do it, I would be very grateful.

Thank you!

**Comment by arvalia** [comment]
All checks have passed... TY @POBrien333 for et-min mentoring!
1st step now is done. Now that there are no more syntax errors, I can start the 2nd step: make beautiful the flow and work on special cases syntax (interviews, graphic archives etc.)

**Pull request closed without merging**